scripts/data/kitti_download.bash

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+DATA_PATH=/data
```

Comment:
Just curious, why /data? Couldn't you do ~/data and then not require permissions?